varia/functions.php

```diff
@@ -140,10 +140,16 @@ function varia_setup() {
 			)
 		);
 
-		// $default_hue     = varia_get_default_hue();
-		// $saturation      = varia_get_default_saturation();
-		// $lightness       = varia_get_default_lightness();
-		// $lightness_hover = varia_get_default_lightness_hover();
+		/*
+		 * Get customizer colors and add them to the editor color palettes
+		 *
+		 * - if the customizer color is empty, use the default
+		 */
+		$colors_array = get_theme_mod('colors_manager'); // color annotations array()
+		$background   = $colors_array['colors']['bg'];   // $config-global--color-background-default;
```

Comment:
One thing I noticed is that this throws errors when I run the theme locally. For example: 

```
Notice: Trying to access array offset on value of type bool in ../wp-content/themes/dotcom/alves/functions.php on line 62

Notice: Trying to access array offset on value of type null in ../wp-content/themes/dotcom/alves/functions.php on line 62
```

Can we wrap these in a conditional to prevent that?

Reply:
Figured out a solution for this and revised the the conditionals in all the child themes so they don’t throw an error on non-dotcom setups, and so they’re a little easier to digest. Also switched to `! empty()` from `! isset()` which is a better [kind of check](https://phppot.com/php/isset-vs-empty-vs-is_null/).